src/Apps/Collect/__tests__/CollectionApp.test.tsx

```diff
@@ -4,6 +4,7 @@ import { graphql } from "react-relay"
 import { Provider } from "unstated"
 import { MockBoot, MockRelayRenderer, renderUntil } from "../../../DevTools"
```

Comment:
This isn't required in test PR's as it can be done in separate focused sweeps, but note that one thing we've starting to do since @ds300's [RFC](https://github.com/artsy/reaction/issues/1166) is turn on the type-checker while working in test files and then before committing turn it off so project doesn't blow up for others. If the line in `tsconfig` is commented out _after_ the compiler starts then type-checking is constrained to only the files open inside of your VSCode.

Reply:
This was completed and merged last night, so disregard!